sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/DecryptParameters.java

```diff
@@ -38,172 +38,187 @@
     private final byte[] authenticationTag;
 
     /**
-     * Factory method to create an instance of {@link DecryptOptions} with the given parameters for
+     * Factory method to create an instance of {@link DecryptParameters} with the given parameters for
      * {@link EncryptionAlgorithm#A128CBC}.
      *
      * @param cipherText The content to be decrypted.
      * @param iv Initialization vector for the decryption operation.
-     * @return The {@link DecryptOptions}.
+     *
+     * @return The {@link DecryptParameters}.
      */
-    public static DecryptOptions createAes128CbcOptions(byte[] cipherText, byte[] iv) {
-        return new DecryptOptions(EncryptionAlgorithm.A128CBC, cipherText, iv, null, null);
+    public static DecryptParameters createAes128CbcParameters(byte[] cipherText, byte[] iv) {
+        return new DecryptParameters(EncryptionAlgorithm.A128CBC, cipherText, iv, null, null);
```

Comment:
We both had a conversation a while ago involving @JonathanGiles about the naming we would follow because some of our APIs were already capitalized using camelCase. In the end we went this route to stay consistent with those older APIs. We can change the capitalization of parameters to be correct, but we cannot change methods, so I think these getters and setters will have to stay that way :/

Reply:
Oh, that's right. Staying consistent with yourselves is better, I agree. Please disregard the other similar comments.